database/seeders/DatabaseSeeder.php

```diff
@@ -19,7 +19,7 @@ class DatabaseSeeder extends Seeder
     public function run(): void
     {
 
-        $course = Course::factory()
+        $course = Course::factory(10)
```

Comment:
There's no need to make changes here. Please remove changes from the Database seeder